students/yushus/session02/series.py

```diff
@@ -36,12 +17,21 @@ def sum_series(n, n0=0, n1=1):
 
     if n0 == 2 and n1 == 1, the result is the Lucas series
     """
+
     if n0 == 0 and n1 == 1:
-        return fibonacci(n)
+        if n <= 1:
+            return n
+        else:
+            return (sum_series(n-1, n0, n1) + sum_series(n-2, n0, n1))
     elif n0 == 2 and n1 == 1:
-        return lucas(n)
+        if n == 0:
+            return 2
+        elif n == 1:
+            return 1
+        else:
+            return (sum_series(n-1, n0, n1) + sum_series(n-2, n0, n1))
     else:
-        raise "Not the right format!!!"
+        raise "Function not defined"
```

Comment:
Still not correct... `sum_series()` is defined for any input values of `n0` and `n1`. You really don't need to check if `n0==0 and n1==1` or `n0==2 and n1==1`. Just forget about that checking, and return `n0` if `n==0`, `n1` if `n==1`, and the recursive result. Note that you have repeated code that's not needed and should be avoided.